ui/shared/DropdownMenuItems.jsx

```diff
@@ -33,14 +33,12 @@ const DropdownMenuItems = ({
     {pinned.length > 0 &&
       options
```

Comment:
As you said above, this doesn't show the pinned items in the defined order. The fix can be included in this PR